frontend/src/pages/premium/waitlist.page.tsx

```diff
@@ -0,0 +1,240 @@
+import { NextPage } from "next";
+import { Localized, useLocalization } from "@fluent/react";
+import styles from "./waitlist.module.scss";
+import { Layout } from "../../components/layout/Layout";
+import { Button } from "../../components/Button";
+import { FormEventHandler, useEffect, useState } from "react";
+import { getLocale } from "../../functions/getLocale";
+import { toast } from "react-toastify";
+import { getRuntimeConfig } from "../../config";
+import { CountryPicker } from "../../components/waitlist/countryPicker";
+import { useRuntimeData } from "../../hooks/api/runtimeData";
+import { LocalePicker } from "../../components/waitlist/localePicker";
+import { useUsers } from "../../hooks/api/user";
+
+/** These are the languages that marketing can send emails in: */
+const supportedLocales = [
```

Comment:
Question: Is there any way to query this data from a dynamic source? I would hate to have lag between this static list and the languages they support. 

Alternatively, do we have a documented (internally) point of contact to confirm this supported language list periodically?

Reply:
Good point; I don't think this list will change that often, but when it does, it's best if we stay aligned. I've asked Tony about this, and about what to do for countries we already support. I'm going to merge this in the meantime though, so that we can test whether the data ends up in the right systems - can fix the rest up in follow-up PRs.